src/lib/components/topBar/TopBar.svelte

```diff
@@ -0,0 +1,204 @@
+<script lang="ts">
+	import TopBarBtn from "$lib/components/topBar/TopBarBtn.svelte";
+	import DropDownBtn from "$lib/components/topBar/DropDownBtn.svelte";
+	import { load } from "$lib/classes/project";
+	import * as Icon from "svelte-google-materialdesign-icons";
```

Comment:
according to our svelte expert, this is not good for the bundling, instead, we should only import the ones we use

Reply:
i only meant to highlight line 5 xd